docs/case-studies/rwanda.md

```diff
@@ -0,0 +1,34 @@
+<h1>Rwanda - Case Study</h1>
+
+## Link to platform
+[https://sustainabledevelopment-rwanda.github.io/](https://sustainabledevelopment-rwanda.github.io/)
+
+## Why was Open SDG chosen?
+We chose this platform because it gave us country ownership and the ability to customise the site, as well as providing key features that we need. The fact it is an open source, it allows the Rwanda SDGs reporting team to link up with others and share knowledge as we network. 
+
+## Which organisation was responsible for choosing and implementing the platform?
+The National Institute of Statistics Rwanda (NISR) 
+
+## How did you set up your country reporting based on Open SDG?
+We have a strategic partnership with the UK Office for National Statistics (ONS) and they worked with the Centre for Open Data Enterprise (CODE) to set up the platform for us. The actual switch of our live site to Open SDG was done during a workshop with ONS in Rwanda.
+
+## How long did it take to set up the platform?
+Adopting the SDG open data did not take long. Just one day to set up the platform. But its updation is ongoing. 
+
+## How do you upload data and metadata into the platform?
+We have data from previous systems (eg DevInfo) as Excel files and receive data from MDAs in various formats. ONS ran a training session and helped us use Excel and scripts (Python and VBA) to convert the data and metadata into the required format and troubleshoot any errors during the upload process.
+
+## How is the platform hosted? (GitHub, local server, other platform)
+Currently the �staging� and public �production� versions are both hosted on GitHub.

```

Comment:
@Norric1Admin When I try to build locally it complains about these smart-quotes. To be safe could we change to regular quotes?

Reply:
Thanks - I didn't notice those. Amended and ready to go.